src/transforms.jl

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
I prefer `transform.jl` for the filename because with `transforms.jl` I might expect subtypes of `Transform` to be defined here.